convert.go

```diff
@@ -94,7 +94,7 @@ func InterfaceToVariable(input interface{}) (ast.Variable, error) {
 		}
 
 		return ast.Variable{
-			Type:  ast.TypeList,
+			Type:  ast.TypeList{ast.TypeAny},
```

Comment:
I expect we need to do something a little more elaborate here to correctly infer the appropriate type, but I'm not sure what exactly this codepath is used for so not sure what the most appropriate behavior would be.